platformio.ini

```diff
@@ -641,3 +641,9 @@ lib_deps =
   ${esp32.lib_deps}
   TFT_eSPI @ ^2.3.70
 board_build.partitions = ${esp32.default_partitions}
+
+[env:esp32-c3-12f]
+extends = env:esp32c3
+board = esp32-c3-12f
+platform = espressif32@5.2.0
```

Comment:
Is this strictly needed? Will it also work with `platform = espressif32@5.1.1` ?
I'm asking because espressif32@5.1.1 has been tested also with other new boards, and there is currently more confidence in the 5.1.1.

Reply:
Yes, that is true. C3 is working for me with 5.1.1 using build environment present in WLED.